packages/api/endpoints/granules.js

```diff
@@ -125,7 +127,12 @@ async function put(req, res) {
       return res.boom.conflict(message);
     }
 
-    await granuleModelClient.move(granule, body.destinations, process.env.DISTRIBUTION_ENDPOINT);
+    await moveGranule(
```

Comment:
The crux of this change is moving the 'move' logic entirely out of the API granule model, re-using the underlying ingest/model code as appropriate.